.github/workflows/check-versions.yaml

```diff
@@ -32,6 +32,30 @@ jobs:
           ./.github/scripts/update-versions.sh
           git diff
 
+      - name: Setup go
+        uses: actions/setup-go@v4.0.0
+
+      - name: Setup crane
+        uses: imjasonh/setup-crane@v0.1
+
+      - uses: actions/setup-python@v4
```

Comment:
Lock on the patch release like we do with other actions.

Reply:
For crane, they only have major.minor